### ISSUE TITLE
SPI PDM for SAMD21 boards

### ISSUE THREAD

**Issue by lxschwalb** [adafruit/Adafruit_ZeroPDM#7]
- Arduino board:  Itsybitsy M0

- Arduino IDE version (found in Arduino -> About Arduino menu):  PlatformIO in VS Code

- List the steps to reproduce the problem below (if possible attach a sketch or
  copy the sketch code in too): Try to compile the spipdm_analogout example for the itsybitsy M0. It fails due to line 7 "Adafruit_ZeroPDMSPI pdmspi(&PDM_SPI);"

After some digging I found that line 8 in "Adafruit_ZeroPDMSPI.h" is:

#if defined(__SAMD51__)

Does this mean that the SPI PDM code is actually written for the SAMD51 boards and not the SAMD21 boards? If so, it would be great to mention that in the readme.md

**Comment by ladyada** [comment]
correct! we only use the code in one place (Edge Badge) which is SAMD51
please submit any update for the readme you'd like as a PR and we'll merge it :)
